### PR TITLE
Vault config

### PR DESCRIPTION
**Pull request by ianaya89**

### CODE REVIEW

File: generators/app/templates/_package.json [hunk 1]
```diff
@@ -43,7 +44,7 @@
     "tribune": "^1.0.1",
     "winston": "^2.1.1",
     "winston-child-logger": "0.0.1",
-    "winston-sentry-transport": "0.0.1"
+    "winston-sentry-transport": "0.0.1",
```
Comment (ndelvalle) [inline generators/app/templates/_package.json]: I think the `,` at the end will generate an error

**Comment by ndelvalle** [comment]
Should we add the MAINTENERS file here?, I think not having it makes it more generic. Also it would be cool to open source it. I know is public already, but maybe we can make it a little bit more generic, add a cool readme and make it ready to go :).

**Comment by ianaya89** [comment]
@ndelvalle I kept the file but without maintainers to make more OSS friendly.

**Comment by RobertWHurst** [comment]
For the open source repos I think we should use GitHub's maintainers group for each repo

**Comment by RobertWHurst** [comment]
LGTM 👍

**Comment by ianaya89** [comment]
LGTM

**Comment by ndelvalle** [comment]
LGTM

**Pull request merged**